README.rst

```diff
@@ -553,6 +553,20 @@ Choose from the list of available rules:
 
   *Risky rule: risky if the ``ereg`` function is overridden.*
 
+* **error_suppression** [@Symfony:risky]
+
+  Error control operator should be added to deprecation notices and/or
+  removed from other cases.
+
+  *Risky rule: adding and/or removing ``@`` might cause changes to code behaviour.*
+
+  Configuration options:
+
+  - ``mute_deprecation_error`` (``false``, ``true``): whether to add ``@`` in deprecation
+    notices; defaults to ``true``
+  - ``noise_remaining_usages`` (``false``, ``true``): whether to remove ``@`` in remaining
```

Comment:
If I would only read option, then I understand `noise_remaining_usages=yes` to actually add `@`.
Is it really adding them ?

Reply:
How come? `@` is known a silence operator, so "noising" would naturally mean removing it?

I've chosen the names "mute" and "noise" as they are opposite words and to have an order (which is alphabetical by option name) where the "remaining" option is second.

Reply:
my bad, ignore :+1: